### ISSUE TITLE
Random BB Error

### ISSUE THREAD

**Issue by Solo-** [JoSchaap/GoT_Wasteland_V2.Stratis#103]
Everytime a mission starts that have advanced rifles in it like the Lynx and GM6 we see this error, however those rifles are in there, do not know if the new Titan launchers are supposed to be in there as well, as those are not in the crates.

![2013-08-03_00005](https://f.cloud.github.com/assets/4993559/907194/9e97ea96-fd16-11e2-9703-93bacaf2b516.jpg)

**Comment by AgentRev** [comment]
This error is showing because `fn_refillbox` is not compiled on clients. Jo, you shouldn't use `fn_vehicleInit` for calling server-only functions.

Or, you could be lazy and add `if (!isNil 'fn_refillbox') { ... }`, that would work too.

**Comment by JoSchaap** [comment]
the outpost should be serversided. but youre right about the fn_vehicleinit :)

I  didnt even notice it was in there..  

_note to self_ https://github.com/JoSchaap/GoT_Wasteland_V2.Stratis/blob/master/server/functions/createOutpost.sqf#L62

**Comment by AgentRev** [comment]
I've read thru your outpost config files, and none of the init functions need to be broadcast via the network manually.

If you want to fix it the proper way, first you gotta browse thru your files in `server/missions/outposts`, and change quotes to curly braces for last parameters (if not empty) of all lines. Second, you gotta add an underscore to all `this` keywords. Example:

```
["Box_East_WpsSpecial_F",[-6.74817,7.19141,0],180,1,0,"[this,'mission_Main_A3snipers'] call fn_refillbox"],
```

to

```
["Box_East_WpsSpecial_F",[-6.74817,7.19141,0],180,1,0,{[_this,'mission_Main_A3snipers'] call fn_refillbox}],
```

Then finally, you gotta change L62 in `createOutpost.sqf` to

```
if (!isNil "_vehicleinit") then { _newObj call _vehicleinit };
```

**Comment by JoSchaap** [comment]
Oh its the dreaded underscores again.. some day.. i'll learn..

its because i test scripts in the editor where its clientsided... i keep forgetting to add those..

**Comment by AgentRev** [comment]
Underscores were not a problem, since `fn_vehicleInit` already adds those automatically, but they are needed if using `call` directly.

**Comment by JoSchaap** [comment]
will be fixed in v2.4
